Development/nmos/system_resources.cpp

```diff
@@ -1,21 +1,37 @@
 #include "nmos/system_resources.h"
 
+#include "cpprest/json_validator.h"
 #include "nmos/is09_versions.h"
+#include "nmos/json_schema.h"
 #include "nmos/json_fields.h"
 #include "nmos/resource.h"
 
 namespace nmos
 {
     nmos::resource make_system_global(const nmos::id& id, const nmos::settings& settings)
     {
-        return{ is09_versions::v1_0, types::global, make_system_global_data(id, settings), true };
+        const auto version = is09_versions::v1_0;
+        const auto data = make_system_global_data(id, settings);
+
+        // ensure system global data is correctly formated
```

Comment:
Typo

Reply:
ensure system global data is correctly `formatted`.